tools/bytes-in-block/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+)
+
+// Stats holds the stats portion of each block
+type Stats struct {
+	BytesInBlock int64 `json:"bytes_in_block"`
+	BlockTime    int64 `json:"block_time"`
+}
+
+// Block represents the entire block from the JSON array
+type Block struct {
+	ID     int   `json:"id"`
+	Height int   `json:"height"`
+	Stats  Stats `json:"stats"`
+}
+
+func main() {
+	start := 4170798
+	end := 4170898
+	limit := 100
+
+	var totalBlocks int64
+	var sumBytes int64
+	var sumBlockTime int64
+
+	// We'll loop offset in increments of `limit`.
+	// For each iteration, we do a GET request with the given offset + limit=100
+	// until we've covered the range [start, end).
+	for offset := start; offset <= end; offset += limit {
```

Comment:
[nit][optional]
rename offset and limit to more understandable names like: `blockRangeStart`, `numberOfBlocks` or similar